doc/mcuboot/readme-ncs.rst

```diff
@@ -3,14 +3,30 @@
 Using MCUboot in nRF Connect SDK
 ################################
 
-See :doc:`readme-zephyr` for general information on how to integrate MCUboot with Zephyr.
+MCUboot can be used in the |NCS| in one of the following roles:
 
-The nRF Connect SDK provides additional functionality that is available when MCUboot is included.
-This functionality is implemented in the files in the :file:`modules/mcuboot` subfolder in the `sdk-nrf`_ repository.
+* As an :ref:`Immutable first-stage bootloader <immutable_bootloader>` that can perform device firmware updates to the application.
+* As an :ref:`Upgradable second-stage bootloader <upgradable_bootloader>` that can perform device firmware updates to both itself and the application.
+
+The MCUboot files in the |NCS| are located in the :file:`modules/mcuboot` subfolder in the `sdk-nrf`_ repository.
```

Comment:
Should this be clarified a bit to say something like MCUboot-specific integration files for NCS are in this subfolder? The current form sounds a lot like MCUboot itself is located there, but it's a separate module.

Reply:
right, fixed